### ISSUE TITLE
handlers.test: %1 is not a valid Win32 application.

### ISSUE THREAD

**Issue by KatamariJr** [onsi/ginkgo#653]
When trying to run any test using `ginkgo -r`, I get the following failure message

```
Failed to run test suite!
        fork/exec C:\Users\agreen\AppData\Local\Temp\ginkgo021687811\handlers.test: %1 is not a valid Win32 application.
```

I usually run ginkgo from my command line on windows 10 (64 bit) but then this suddenly started happening.

**Comment by blgm** [comment]
Hi @KatamariJr, which version of Ginkgo is being used? Is it possible that the version has changed since the last time it was successful?

**Comment by KatamariJr** [comment]
Hello @blgm, I was using v1.10.3 when this was posted, just tried updating to v1.12.0. Not sure what version I had been using, or if it had updated without my knowledge. After updating, I was able to run my tests once, but then any subsequent attempts would result in the same error as before. This also occurs when I run `go test ./...`, which I assume is because go test just runs ginkgo.

**Comment by blgm** [comment]
This isn't an area of Ginkgo that I'm familiar with, but this is what I've worked out so far:

- At initialisation, Ginkgo makes a list of all the suites that it needs to compile
- It then compiles each test suite in an independent temporary directory. I think that's where `C:\Users\agreen\AppData\Local\Temp\ginkgo021687811\handlers.test` is coming from. Is it correct to assume that there is a test suite called `handlers`?
https://github.com/onsi/ginkgo/blob/f8ab89d658e4fa0e92bdc1023b9a884fcd3faefe/ginkgo/testrunner/test_runner.go#L56
- Ginkgo then attempts to execute the compiled test
- Windows cannot execute the compiled test

To move the investigation forward, it would be useful to examine what Ginkgo has compiled. In order to do this you'd need to temporarily comment out the cleanup line here:
https://github.com/onsi/ginkgo/blob/f8ab89d658e4fa0e92bdc1023b9a884fcd3faefe/ginkgo/testrunner/test_runner.go#L251

I replaced it with the line `fmt.Printf("*** %+v\n", filepath.Dir(t.compilationTargetPath))` so that I could find the directory.

To rebuild Ginkgo, run `go install github.com/onsi/ginkgo/ginkgo`

1) Is there a file in the temporary directory?
2) Is it executable?
3) Is there anything else of note?
